### PR TITLE
added a navbar to app

### PR DESCRIPTION
**Pull request by PLN2020**
what do you guys think?

### CODE REVIEW

**Comment by PLN2020** [comment]
image
here is a screenshot of the navbar

**Comment by paulxcodes** [comment]
Please remove messaging function this is a personal liability to enable these people

**Comment by paulxcodes** [comment]
Rename "messages" to "history"

**Comment by PLN2020** [comment]
> Rename "messages" to "history"

ok will do

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -12,6 +12,7 @@
   </head>
 
   <body>
+    <%= render 'shared/navbar' %>
```
Comment (paulxcodes) [inline app/views/layouts/application.html.erb:15]: Very good style!

**Review by paulxcodes — APPROVED** [review]
LGTM!

**Comment by paulxcodes** [comment]
@PLN2020 - can you use this template for the navbar? We need to have have the piece of code in there for signing in: https://github.com/lewagon/awesome-navbars/blob/master/templates/_navbar_wagon.html.erb

**Comment by PLN2020** [comment]
> @PLN2020 - can you use this template for the navbar? We need to have have the piece of code in there for signing in: https://github.com/lewagon/awesome-navbars/blob/master/templates/_navbar_wagon.html.erb

I've used the template but can't preview it because devise hasn't been setup yet so a routing error will display. Please review ty

**Comment by paulxcodes** [comment]
> > @PLN2020 - can you use this template for the navbar? We need to have have the piece of code in there for signing in: https://github.com/lewagon/awesome-navbars/blob/master/templates/_navbar_wagon.html.erb
> 
> I've used the template but can't preview it because devise hasn't been setup yet so a routing error will display. Please review ty

Perfect

**Pull request merged**